docker/api-gateway/conf.d/dev.conf

```diff
@@ -7,4 +7,10 @@ server {
         proxy_set_header Host            $host;
         proxy_set_header X-Forwarded-For $remote_addr;
     }
+
+    location /api/job {
+            proxy_pass                       http://host.docker.internal:8090/api/job;
```

Comment:
In `dev` environment we should map to a different port since 8090 of the host machine will be occupied by file storage service. So let's use 8091 for job service here.

Reply:
Also don't forget to update `README.md` file with information about new service